AAI/AAIConnectProfile.md

```diff
@@ -399,29 +407,62 @@ the Broker.
     except for accommodating for `exp` timestamps to be represented as
     indicated above.
 
-#### Conformance for Claim Clearinghouses (consuming Access Tokens to give access to data)
+#### Conformance for Token Container Issuers
```

Comment:
This will need to be backwards compatible, and need more version controls in place to bring it to spec. I'd recommend we save this for a separate PR after we rework the "what is a v1.0 passport?" clarification landed.